spec.md

```diff
@@ -288,6 +288,31 @@ encapsulated within the `data` attribute.
 * Constraints:
   * OPTIONAL
 
+# Privacy & Security Considerations
+Interoperability is the primary driver behind this specification, enabling such
+behavior requires some information to be made available *in the clear* resulting
+in the potential for information leakage.
+
+The following should be considered to prevent inadvertent leakage especially when
```

Comment:
Avoid "should", perhaps just "Consider the following to prevent...."

Reply:
Done